### ISSUE TITLE
What's the motivation to use BatchNorm instead of Layer Norm?

### ISSUE THREAD

**Issue by maudzung** [PointsCoder/VOTR#5]
Hi @PointsCoder ,
Thanks for your great work.
Can I ask you why did you use Batch Norm in your paper instead of Layer Norm? Did you compare their results? I couldn't find their comparison in the paper.
Thank you so much!

**Comment by PointsCoder** [comment]
@maudzung LayerNorm achieves around 78 mod. AP for VoTr-SSD on KITTI with 8 GPUs (refer to votr_new_backbone.py), while BatchNorm achieves 78.2 mod. AP with 1 GPU. My initial conclusion is that those two norm have comparable performance, but I didn't try LayerNorm on 1 GPU. I may try it later if I have time.

**Comment by maudzung** [comment]
Thank you
